### ISSUE TITLE
Incremental Build?

### ISSUE THREAD

**Issue by ickc** [fauno/jekyll-pandoc-multiple-formats#23]
It seems that incremental build is not working for other file types, say, PDF?
In the being of the build, all pdf are rendered again regardless if they have been rendered already (and hasn't changed). When a file is undated and watched, the HTML is generated but the PDF is not.

**Comment by fauno** [comment]
nope, sorry, it's something we'd like to add but haven't come to it
yet.  any ideas on how to implement it?

-- 
@y

**Comment by fauno** [comment]
i'd have to review jekyll's implementation of incremental builds but i'd do something like make: check dependencies modification times and if any is newer than target, then build it.  though we'd have to check pandoc's flags for included files
